packages/nimble-components/src/utilities/tests/storybook.ts

```diff
@@ -146,16 +146,19 @@ Overrides of properties are not recommended and are not theme-aware by default.
 ${howToOverride}
 </details>`;
 
-export const usageWarning = (componentName: string): string => `
+export const incubatingWarning = (
+    componentName: string,
+    statusLink: string
+): string => `
```

Comment:
nit / pref: could you make this a configuration object instead? Will make it easier to understand the params if the list grows